### ISSUE TITLE
daily JSON and CSV exports failing

### ISSUE THREAD

**Issue by paregorios** [isawnyu/pleiades-gazetteer#430]
It looks like we haven't had a complete run since 2 July. I'll have to inspect the logs, but if past is prologue, this is likely due to a piece of broken content somewhere.

**Comment by paregorios** [comment]
CSV exports are also affected

**Comment by paregorios** [comment]
KML and RDF/TTL exports seem to have continued to run to completion, so they may be handling whatever the error is more gracefully than our other exports.

**Comment by paregorios** [comment]
Ran the dump script manually and watched it's output. Sure enough, threw a traceback trying to access a location. The culprit seems to have been that the location in question, instead of referencing a published "accuracy assessment" object was referencing an unpublished location object (how on earth could such a thing happen?). In any case, editing the location and using the reference browser to connect to the proper accuracy assessment resolved that traceback.

But, running the dump again manually, there is now another error ...

**Comment by paregorios** [comment]
The next road bump was a published location with a linked accuracy assessment that was not published. These are all very old. Why should they only be throwing errors now?

But now another location with the same weird problem noted above? Is this due to a UUID collision?

**Comment by paregorios** [comment]
I've found a few more of these. I think these data problems (some locations with links to things other than accuracy assessments) have been in place for years, but the dump never tripped over them because we weren't trying to access any data in the linked accuracy assessment. [The code I rolled out last week](https://github.com/isawnyu/PleiadesEntity/blob/7278c7a244f60b4d7c61b9e5b625c81925d503f0/Products/PleiadesEntity/browser/adapters/location.py#L51-L52) does access the "value" field on the accuracy object and the necessary traverse I added to get there is what's throwing this error when it finds a bad link.

**Comment by paregorios** [comment]
Exception handling has been added to the export code for (1) attempts to access unauthorized accuracy assessments and (2) failed attempts to access the "value" field on an accuracy object (i.e., if it's not really an accuracy object and therefore doesn't have a "value" field. These code changes allow the full export to run to completion and log errors to dump.log. This morning's daily run was successful. Also, I have manually addressed all the errors logged, so I think the underlying data problems have all been addressed (these seem all to have dated from a time where we didn't have good filtering on the reference browser for adding the accuracy assessment).
